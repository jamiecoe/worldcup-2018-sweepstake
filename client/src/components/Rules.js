import React from 'react';

const Rules = () => {
  return (
    <div>
      <h2>The rules:</h2>
      <ul>
        <li>Each player gets <strong>2 top</strong>, <strong>2 mid</strong> and <strong>4 low</strong> countries (country rankings are available <a href="http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html" rel="noopener noreferrer" target="_blank">here</a>)</li>
        <li>The player with the country that wins the World Cup, wins the sweep stake</li>
      </ul>
    </div>  
  )
}

export default Rules;
