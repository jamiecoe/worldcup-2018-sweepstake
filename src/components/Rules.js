import React from 'react';

const Rules = () => {
  return (
    <div>
      <h2>The rules:</h2>
      <ul>
        <li>Each person gets <strong>2 top</strong>, <strong>2 mid</strong> and <strong>4 low</strong> countries (country rankings are available <a href="http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html" rel="noopener noreferrer" target="_blank">here</a>)</li>
        <li>Whoever has the country that wins the World Cup will win the sweep stake</li>
      </ul>
    </div>  
  )
}

export default Rules;
