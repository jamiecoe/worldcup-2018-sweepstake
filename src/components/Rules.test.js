import React from 'react';
import { shallow } from 'enzyme';
import Rules from './Rules';

describe('Rules', () => {
  const rules = shallow(<Rules />);

  it('should render properly', () => {
    expect(rules).toMatchSnapshot();
  });
});
