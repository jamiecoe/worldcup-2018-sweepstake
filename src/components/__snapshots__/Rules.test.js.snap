// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rules should render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Rules />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          The rules:
</h2>,
        <ul>
          <li>
                    Each player gets 
                    <strong>
                              2 top
                    </strong>
                    , 
                    <strong>
                              2 mid
                    </strong>
                     and 
                    <strong>
                              4 low
                    </strong>
                     countries (country rankings are available 
                    <a
                              href="http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html"
                              rel="noopener noreferrer"
                              target="_blank"
                    >
                              here
                    </a>
                    )
          </li>
          <li>
                    The player with the country that wins the World Cup, wins the sweep stake
          </li>
</ul>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "The rules:",
        },
        "ref": null,
        "rendered": "The rules:",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              Each player gets 
              <strong>
                            2 top
              </strong>
              , 
              <strong>
                            2 mid
              </strong>
               and 
              <strong>
                            4 low
              </strong>
               countries (country rankings are available 
              <a
                            href="http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html"
                            rel="noopener noreferrer"
                            target="_blank"
              >
                            here
              </a>
              )
</li>,
            <li>
              The player with the country that wins the World Cup, wins the sweep stake
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Each player gets ",
                <strong>
                  2 top
</strong>,
                ", ",
                <strong>
                  2 mid
</strong>,
                " and ",
                <strong>
                  4 low
</strong>,
                " countries (country rankings are available ",
                <a
                  href="http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html"
                  rel="noopener noreferrer"
                  target="_blank"
>
                  here
</a>,
                ")",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Each player gets ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "2 top",
                },
                "ref": null,
                "rendered": "2 top",
                "type": "strong",
              },
              ", ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "2 mid",
                },
                "ref": null,
                "rendered": "2 mid",
                "type": "strong",
              },
              " and ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "4 low",
                },
                "ref": null,
                "rendered": "4 low",
                "type": "strong",
              },
              " countries (country rankings are available ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "here",
                  "href": "http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "here",
                "type": "a",
              },
              ")",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "The player with the country that wins the World Cup, wins the sweep stake",
            },
            "ref": null,
            "rendered": "The player with the country that wins the World Cup, wins the sweep stake",
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            The rules:
</h2>,
          <ul>
            <li>
                        Each player gets 
                        <strong>
                                    2 top
                        </strong>
                        , 
                        <strong>
                                    2 mid
                        </strong>
                         and 
                        <strong>
                                    4 low
                        </strong>
                         countries (country rankings are available 
                        <a
                                    href="http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html"
                                    rel="noopener noreferrer"
                                    target="_blank"
                        >
                                    here
                        </a>
                        )
            </li>
            <li>
                        The player with the country that wins the World Cup, wins the sweep stake
            </li>
</ul>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "The rules:",
          },
          "ref": null,
          "rendered": "The rules:",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                Each player gets 
                <strong>
                                2 top
                </strong>
                , 
                <strong>
                                2 mid
                </strong>
                 and 
                <strong>
                                4 low
                </strong>
                 countries (country rankings are available 
                <a
                                href="http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html"
                                rel="noopener noreferrer"
                                target="_blank"
                >
                                here
                </a>
                )
</li>,
              <li>
                The player with the country that wins the World Cup, wins the sweep stake
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Each player gets ",
                  <strong>
                    2 top
</strong>,
                  ", ",
                  <strong>
                    2 mid
</strong>,
                  " and ",
                  <strong>
                    4 low
</strong>,
                  " countries (country rankings are available ",
                  <a
                    href="http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html"
                    rel="noopener noreferrer"
                    target="_blank"
>
                    here
</a>,
                  ")",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Each player gets ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "2 top",
                  },
                  "ref": null,
                  "rendered": "2 top",
                  "type": "strong",
                },
                ", ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "2 mid",
                  },
                  "ref": null,
                  "rendered": "2 mid",
                  "type": "strong",
                },
                " and ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "4 low",
                  },
                  "ref": null,
                  "rendered": "4 low",
                  "type": "strong",
                },
                " countries (country rankings are available ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "here",
                    "href": "http://www.fifa.com/fifa-world-ranking/ranking-table/men/index.html",
                    "rel": "noopener noreferrer",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": "here",
                  "type": "a",
                },
                ")",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The player with the country that wins the World Cup, wins the sweep stake",
              },
              "ref": null,
              "rendered": "The player with the country that wins the World Cup, wins the sweep stake",
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
